lib/livebook/hubs/dockerfile.ex

```diff
@@ -108,13 +116,30 @@ defmodule Livebook.Hubs.Dockerfile do
     RUN /app/bin/warmup_apps.sh
     """
 
+    startup =
+      if config.cluster == :fly_io do
+        ~S"""
+        # Custom startup script to cluster multiple Livebook nodes
+        RUN printf '#!/bin/bash\n\
+        export ERL_AFLAGS="-proto_dist inet6_tcp"\n\
```

Comment:
Shall we move the static variables out of the script?

Reply:
Is there a specific reason? If we could move them all that would make sense, but since `LIVEBOOK_NODE` is runtime, I would keep together for consistency.

Reply:
The idea was to keep only runtime ones in the runtime script. Also, any thoughts in an approach like this?

```
ENV ERL_AFLAGS "-proto_dist inet6_tcp"
ENV LIVEBOOK_DISTRIBUTION name
CMD [ 'sh', '-c' , 'LIVEBOOK_NODE="${FLY_APP_NAME}-${FLY_IMAGE_REF##*-}@${FLY_PRIVATE_IP}" LIVEBOOK_CLUSTER="dns:${FLY_APP_NAME}.internal" /app/bin/livebook start']
```